connect/api/src/main/java/org/apache/kafka/connect/storage/StringConverter.java

```diff
@@ -79,7 +79,7 @@ public void configure(Map<String, ?> configs, boolean isKey) {
         try {
             return serializer.serialize(topic, value == null ? null : value.toString());
         } catch (SerializationException e) {
-            throw new DataException("Failed to serialize to a string: ", e);
+            throw new DataException(this.getClass() + " Failed to serialize to a string: ", e);
```

Comment:
Using `Class.toString()` method here will output a message like:
```
class org.apache.kafka.connect.storage.StringConverter Failed to serialize to a string: ...
```
First, beginning this exception message with `class` it not capitalized even though it's the start of a sentence, and `Failed` should not be capitalized in the middle of a sentence. Plus, I'm unconvinced that including the package is helpful.

But more importantly, I don't think we should change these exception messages in any of the converters, because we'd have to change them in all converter implementations to remain consistent. While we could do that for AK-provided converters, there are lots of other converters that may have modeled their error messages after these, and would then become inconsistent. Plus, why change all of them when we're catching the exception in the WorkerSinkTask and WorkerSourceTask? Wouldn't it make more sense to change it there so that we don't have to change any converters?